src/Compilers/CSharp/Portable/Symbols/Source/SourceCustomEventAccessorSymbol.cs

```diff
@@ -68,7 +68,7 @@ internal sealed class SourceCustomEventAccessorSymbol : SourceEventAccessorSymbo
             }
             else
             {
-                var bodyOpt = syntax.Body;
+                var bodyOpt = syntax.Body as CSharpSyntaxNode ?? syntax.ExpressionBody;
```

Comment:
Please use `(CSharpSyntaxNode)syntax.Body` rather than `as CSharpSyntaxNode` since the cast cannot fail.